packages/apple-pay-integration/project.json

```diff
@@ -20,5 +20,6 @@
       }
     }
   },
-  "tags": ["scope:integration"]
+  "tags": ["scope:integration"],
+  "types": "index.d.ts"
```

Comment:
I cannot find the doc about `types` here. Could you please explain what this is for?

Reply:
Was a leftover, will remove